src/GenerateKeyboardDoc.test.js

```diff
@@ -0,0 +1,128 @@
+// Generates the keyboard Markdown document
+//
+// Note: this file assumes that the working directory is the project directory.
+//
+// To generate the keyboard Markdown document file:
+//
+// - Set 'generateDoc' to true below
+// - Run this test with: npm test GenerateKeyboardDoc
```

Comment:
Unless we plan to do this as part of our tests in CI, this doesn't need to be a test.

It seems better to convert this to a node script.  You'd then make a separate npm script definition for this, i.e. `npm generateDoc`.

Once it's no longer a test, you could get rid of the `generateDoc` variable.

Reply:
I can see that this is more involved than I'd like because node is not great with imports.  One option would be to move the definitions to a json(5) file, and import that into `KeyboardInputSchemes.js`.

Reply:
(and into your doc generation script).

Reply:
I wonder if I should pull this out as a separate issue. In the current code base, we generate the docs by uncommenting code inside the test - this seemed like a good interim solution.

Reply:
Yeah, this is fine for now, given how much work we'd have to do to do it any other way.